### ISSUE TITLE
Improve error message when the user needs to install a workload

### ISSUE THREAD

**Issue by danroth27** [dotnet/sdk#18119]
If you try to do an action that requires a workload to be installed, you get a rather unhelpful error message that tells you the IDs of the missing workload packs, but not the actual package IDs or install command that you need to run to install the workload:

```
C:\Program Files\dotnet\sdk\6.0.100-preview.4.21255.9\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.ImportWorkloads.targets(24,5): error NETSDK1147: The following workload packs were not installed: Microsoft.NET.Runtime.MonoAOTCompiler.Task Microsoft.NET.Runtime.WebAssembly.Sdk Microsoft.Netcore.App.Runtime.Aot.Cross.browser-wasm Microsoft.NET.Runtime.Emscripten.Python Microsoft.NET.Runtime.Emscripten.Node Microsoft.NET.Runtime.Emscripten.Sdk [C:\Users\WDAGUtilityAccount\Desktop\PictureFixer-main\PictureFixer\Client\PictureFixer.Client.csproj]
```

It's not clear how the user is supposed to figure out what to do to acquire these missing workload packs.

@KathleenDollard @marcpopMSFT @mhutch

**Comment by mhutch** [comment]
It should be suggesting a workload that provides the missing packs /cc @dsplaisted

**Comment by dsplaisted** [comment]
This should be fixed in Preview 5 with #17512

The errors now look like this:

> c:\git\dotnet-sdk2\artifacts\bin\redist\Debug\dotnet\sdk\6.0.100-dev\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.ImportWorkloads.targets(33,5): error NETSDK1147: To build this project, the following workloads must be installed: microsoft-android-sdk-full [c:\git\dotnet-sdk2\artifacts\tmp\Debug\It_should_fai---14415423\WorkloadTest\WorkloadTest.csproj]
> c:\git\dotnet-sdk2\artifacts\bin\redist\Debug\dotnet\sdk\6.0.100-dev\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.ImportWorkloads.targets(33,5): error NETSDK1147: To install these workloads, run the following command: dotnet workload install microsoft-android-sdk-full [c:\git\dotnet-sdk2\artifacts\tmp\Debug\It_should_fai---14415423\WorkloadTest\WorkloadTest.csproj]
